src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -191,6 +231,17 @@ export default {
 			const text = this.genericContentBlock?.bodyCopy ?? '';
 			return text ? richTextRenderer(text) : '';
 		},
+		gridHeroBody() {
+			return ['first-grid-item', 'second-grid-item', 'third-grid-item', 'fourth-grid-item'].map(el => {
+				const block = this.getGridBlock(el);
+				const text = block?.bodyCopy ?? '';
+				if (!text) return false;
```

Comment:
Does this `false` cause an error in the template? I see the template expects `block.text`, which should fail if the block is `false` like in this return. Should there instead be a `.filter` before this map?